pyart/core/radar.py

```diff
@@ -788,6 +788,52 @@ def add_field(self, field_name, dic, replace_existing=False):
         self.fields[field_name] = dic
         return
 
+    def add_filter(self, gatefilter, replace_existing=False,
+                   include_fields=None):
+        """
+        Updates the radar object with an applied gatefilter provided
```

Comment:
Can you add more here, stating that it masks values?

Reply:
Sounds good!